packages/web/src/components/basic/DataController.js

```diff
@@ -46,7 +51,17 @@ class DataController extends Component {
 
 	updateQuery = (defaultSelected = null, props) => {
 		this.locked = true;
-		const query = props.customQuery || DataController.defaultQuery;
+		const { customQuery } = props;
+		const query = customQuery || DataController.defaultQuery;
+
+		const customQueryOptions = customQuery
+			? getOptionsFromQuery(customQuery(defaultSelected, props))
+			: null;
+		this.queryOptions = {
+			...this.queryOptions,
+			...customQueryOptions,
```

Comment:
This won't work if `customQueryOptions` is `null` as per Line 59.

Reply:
@metagrover Can you please describe what'll be the problem if `customQueryOptions` is `null`, because if it's `null` then still the `queryOptions` will be an empty object.

Reply:
If customQuery is null, we can't spread it - `{ ...null }` won't work.

Reply:
My bad. This works!